config/products.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    // Set to true to handle automatically pQty/pQtyUnlim for products with variations
+    'autoQuantityIfVariations' => false,
```

Comment:
The option is turned off by default. If we want to change this, we simply have to change this value.